contrib/platform/test/com/sun/jna/platform/unix/X11Test.java

```diff
@@ -76,6 +77,24 @@ public void testXFetchName() {
         }
     }
 
+    public void testXSetWMProtocols() {
+        X11.Atom[] protocols = new X11.Atom[]{ X11.INSTANCE.XInternAtom(display, "WM_DELETE_WINDOW", false) };
+        int status = X11.INSTANCE.XSetWMProtocols(display, root, protocols, 1);
+        assertEquals("Bad status for XSetWMProtocols", 1, status);
+    }
+
+    public void testXGetWMProtocols() {
+        X11.Atom[] setProtocols = new X11.Atom[]{ X11.INSTANCE.XInternAtom(display, "WM_DELETE_WINDOW", false) };
+        X11.INSTANCE.XSetWMProtocols(display, root, setProtocols, 1);
```

Comment:
With regard to the hard coded count value please see comment for line 82, I'd like to see a test with multiple values.